src/test/java/edu/harvard/iq/dataverse/export/ddi/dataset-create-new-all-ddi-fields.json

```diff
@@ -971,9 +971,12 @@
           },
           {
             "typeName": "collectionMode",
-            "multiple": false,
+            "multiple": true,
             "typeClass": "primitive",
-            "value": "CollectionMode"
+            "value": [
+              "CollectionMode"
+            ]
```

Comment:
Does this mean that the native JSON format is changing? If so we should indicate this under "Backward Incompatibilities" in the release notes.

Reply:
I believe the answer is "yes". (I'm not 100% sure, tbh. the above change in the .json file was needed for the tests to pass. There is a small chance that a real json import would still work; and the single primitive value would be accepted... I should've tested that). 

OK, I'll look for an example of an advertised backward incompatibility in the past release notes.

Reply:
Thanks. If it helps, the 5.10 release notes already has a "Backwards Incompatibilities" section.

Reply:
You're right, thanks!

Reply:
It is an incompatibility, yes.